### PR TITLE
Dev libs/libsass version bump

### PR DESCRIPTION
**Pull request by AndrewAmmerlaan**
version bump and clean up old masked version

### CODE REVIEW

**Comment by chewi** [comment]
Is it actually fixed? The upstream issue is still open.

**Comment by AndrewAmmerlaan** [comment]
> Is it actually fixed? The upstream issue is still open.

I read in the changelog that they fixed a couple of issues causing memory leaks, so I assumed that this was fixed, but now I'm not sure.
I don't actually use this package myself (heck I'm not even sure what it does to be honest), so I'm not sure how to test for it. (This should really be maintained by someone who knows more about this package, I feel like I'm completely in the dark here)

**Comment by chewi** [comment]
I can't remember whether I reproduced it myself and I can't try right now but building x11-themes/arc-theme with `USE=gtk3` should do it.

**Comment by chewi** [comment]
As for what it does, I believe it compiles [Sass](https://sass-lang.com) into plain CSS.

**Comment by AndrewAmmerlaan** [comment]
> I can't remember whether I reproduced it myself and I can't try right now but building x11-themes/arc-theme with USE=gtk3 should do it.

Nope, nope, definitely not fixed

That filled up my RAM *really* fast

I'll edit my PR to keep the mask

**Comment by AndrewAmmerlaan** [comment]
> I'll edit my PR to keep the mask

Fixed, it's just the version bump now

File: profiles/package.mask [hunk 1]
```diff
@@ -508,9 +508,9 @@ net-misc/wicd
 
 # Andrew Ammerlaan <andrewammerlaan@riseup.net> (2020-01-26)
 # Proxy Maintainers <proxy-maint@gentoo.org>
-# v3.6.3 uses huge amounts of memory: Bug #705682
+# v3.6.4 uses huge amounts of memory: Bug #705682
 # v3.6.1 is the latest version without this issue
-=dev-libs/libsass-3.6.3
+=dev-libs/libsass-3.6.4
```
Comment (chewi) [inline profiles/package.mask:513]: We don't know how long this is going to go on for and we want to keep 3.6.3 masked. I would have expected 3.6.4 to fix it but here we are so best make it `>=dev-libs/libsass-3.6.2`.

Comment (AndrewAmmerlaan) [reply profiles/package.mask]: Good idea,

I thought I read somewhere that the next release will be 4.0 so hopefully it will be fixed in that release.

**Pull request closed without merging**